core-client/src/main/java/org/glassfish/jersey/client/authentication/HttpAuthenticationFilter.java

```diff
@@ -419,4 +431,23 @@ static Credentials getCredentials(ClientRequestContext request, Credentials defa
             return specificCredentials != null ? specificCredentials : defaultCredentials;
         }
     }
+
+    private static void discardInputAndClose(InputStream is) {
```

Comment:
This discardInputAndClose is once again in HttpAuthenticationFilter in PR3859. Can you put it to the AuthenticationUtil as discussed in PR 3862?

Reply:
Yes. Done.